cmd/run_controller/main.go

```diff
@@ -92,37 +97,55 @@ func init() {
 }
 
 func main() {
-	defer klog.Flush()
-
```

Comment:
From the ⁠[klog docs](https://pkg.go.dev/k8s.io/klog/v2#section-documentation):

> Log output is buffered and written periodically using Flush. Programs should call Flush before exiting to guarantee all log output is written.

Therefore, restore this. It's for the _normal_ program termination (not via os.Exit()).

Reply:
Done b7ba06e8160ab4760f214fda15639a6773ceca5c